<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>767</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>484</width>
    <height>767</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>484</width>
    <height>767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-size: 18px;
	color: #323232;
	font-weight: bold;
}
QGroupBox::title{
	font-size: 13px;
	font-weight: bold;
	subcontrol-origin:margin;
	subcontrol-position:top left;
	padding:0 3px;
}
QGroupBox {
	margin-top: 20px;
	border: none;
}
QPushButton {
    height: 1.66em;
    padding: 2px 15px;
	border: 2px solid black;
    border-radius: 10px;
	cursor: pointer;
}
QPushButton:hover {
    border: 2px solid #00b8f6;
}
QLineEdit {
    border: 1px solid #fb7299;
    border-radius: 10px;
}
QLineEdit:focus {
    border: 1px solid skyblue;
}
QProgressBar { border: 2px solid grey;
 border-radius: 10px; 
 background-color: #FFFFFF;
 text-align: center;
}
QProgressBar::chunk {background:QLinearGradient(x1:0,y1:0,x2:2,y2:0,stop:0 #666699,stop:1  #00aeec);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>登录设置</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="loginBtn">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>登录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="outBtn">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>登出</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#groupBox {
	border: 1px solid gray;
}</string>
       </property>
       <property name="title">
        <string>下载设置</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="font">
           <font>
            <family>微软雅黑 Light</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string>基本设置</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>-1</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>视频格式</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="videoFormat">
                 <item>
                  <property name="text">
                   <string>mp4</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>msa</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>-1</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>清晰度</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="definition">
                 <item>
                  <property name="text">
                   <string>360p</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>480p</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>720p</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>1080p</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>-1</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>视频地址</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="videoUrl"/>
               </item>
               <item>
                <widget class="QPushButton" name="pushBtn">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>-1</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>加入队列</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="downloadBtn">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>-1</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>一键下载</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="font">
             <font>
              <family>微软雅黑 Light</family>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="title">
             <string>特殊设置</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QCheckBox" name="onlyAudioBtn">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>-1</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>只下载音频</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>-1</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>音频格式:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="audioFormat">
               <item>
                <property name="text">
                 <string>mp3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>aac</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>msa</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#groupBox_2 {
	border: 1px solid gray;
}</string>
       </property>
       <property name="title">
        <string>任务队列</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QListWidget" name="taskWidget"/>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="font">
             <font>
              <family>等线 Light</family>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="title">
             <string>操作</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="deleteTask">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>-1</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>删除</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="upTask">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>-1</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>提升</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="downTask">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>-1</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>下降</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>进度追踪</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
